BufferListAcceptedTypes.d.ts

```diff
@@ -0,0 +1,10 @@
+import BufferList = require('./BufferList');
```

Comment:
I was aiming to not create a different file just for this... maybe until we'll have to support CJS imports, we'll have not other choices...?

Reply:
This is cleaner, and also helps when directly importing `bl/BufferList`.

Reply:
I didn't want this file either, but there were no other way to access that Type without exposing it elsewhere, and also the other way I was getting a lot of problems.